test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -139,7 +139,7 @@ def test_successful_refund
   end
 
   def test_successful_partial_refund
-    gateway = SumUpGateway.new(fixtures(:sum_up_account_for_successful_purchases))
+    gateway = SumUpGateway.new(fixtures(:sum_up_successful_purchase))
```

Comment:
❓ Question
I know you need a 3ds account coming from Europe. Is it possible to use the same 3ds account credentials here instead of this one?

Reply:
No, it is because the account for `sum_up_successful_purchase` return a successful purchase, `sum_up_3ds` return a pending purchase and we can't `refund` a pending purchase.